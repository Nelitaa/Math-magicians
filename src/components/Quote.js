import React from 'react';

function Quote() {
  return (
    <section>
      <p className="quote-text">
        Mathematics is not about numbers, equations, computations,
        or algorithms: it is about understanding.
        <strong>â€” William Paul Thurston</strong>
      </p>
    </section>
  );
}

export default Quote;
