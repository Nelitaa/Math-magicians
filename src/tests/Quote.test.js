import { render } from '@testing-library/react';
import Quote from '../components/Quote';

test('renders Quote component', () => {
  const { asFragment } = render(<Quote />);
  expect(asFragment()).toMatchSnapshot();
});
