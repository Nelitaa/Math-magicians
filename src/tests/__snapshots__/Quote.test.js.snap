// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Quote component 1`] = `
<DocumentFragment>
  <section>
    <p
      class="quote-text"
    >
      Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding.
      <strong>
        â€” William Paul Thurston
      </strong>
    </p>
  </section>
</DocumentFragment>
`;
